omero/developers/testing.txt

```diff
@@ -401,11 +257,38 @@ There are a small number of custom markers defined:
       def testIntermittent():
           self.somePotentiallyIntermittentCall()
 
+.. _using-markers-in-python-tests:
+
+Using markers in OmeroPy tests
```

Comment:
And this section feels like it partly belongs in Writing as well as here. I don't know if there is an easy way for it to be split?